jib-core/src/main/java/com/google/cloud/tools/jib/http/BlobHttpContent.java

```diff
@@ -20,16 +20,22 @@
 import com.google.cloud.tools.jib.blob.Blob;
 import java.io.IOException;
 import java.io.OutputStream;
+import java.time.Duration;
+import java.util.function.Consumer;
 
 /** {@link Blob}-backed {@link HttpContent}. */
 public class BlobHttpContent implements HttpContent {
 
   private final Blob blob;
   private final String contentType;
+  // TODO: Refactor into BlobPushMonitor or something.
+  private final Consumer<Long> sentByteCountConsumer;
+  private final Duration delayBetweenCallbacks = Duration.ofMillis(100);
```

Comment:
This is more like a `static final` and all caps (The class `Duration` is immutable.)

Reply:
I intend to actually refactor this and `sentByteCountConsumer` to a `BlobPushMonitor` so that the actual delay can be configured from the outside and not intending `delayBetweenCallbacks` to remain a constant.